raiden/tests/mocked/test_matrix_transport.py

```diff
@@ -103,9 +115,90 @@ def mock_get_user_presence(self, user_id: str):
     monkeypatch.setattr(transport._raiden_service, "on_messages", mock_on_messages)
     monkeypatch.setattr(GMatrixClient, "get_user_presence", mock_get_user_presence)
 
+    monkeypatch.setattr(transport._client.api, "leave_room", lambda room_id: None)
+    monkeypatch.setattr(transport._client, "sync_token", "already_synced")
+
     return transport
 
 
+def create_new_users_for_address(signer=None, number_of_users=1):
+    users = list()
+    if signer is None:
+        signer = make_signer()
+
+    for i in range(number_of_users):
+        user_id = f"@{signer.address_hex.lower()}:server{i}"
+        signature_bytes = signer.sign(user_id.encode())
+        signature_hex = encode_hex(signature_bytes)
+        user = User(api=None, user_id=user_id, displayname=signature_hex)
+        users.append(user)
+    return users
+
+
+@pytest.fixture()
+def room_with_members(mock_raiden_service, partner_config_for_room):
+
+    number_of_partners = partner_config_for_room["number_of_partners"]
+    users_per_address = partner_config_for_room["users_per_address"]
+    number_of_base_users = partner_config_for_room["number_of_base_users"]
+    room_members = list()
+    base_users = create_new_users_for_address(mock_raiden_service.signer, number_of_base_users)
+    room_members.extend(base_users)
+
+    for _ in range(number_of_partners):
+        users = create_new_users_for_address(number_of_users=users_per_address)
+        room_members.extend(users)
+
+    room_id = "!roomofdoom:server"
+    room = Room(client=None, room_id=room_id)
+    for member in room_members:
+        room._mkmembers(member)
+
+    return room, number_of_partners > 1
+
+
+@pytest.mark.parametrize(
+    "partner_config_for_room",
+    [
+        {"number_of_partners": 1, "users_per_address": 1, "number_of_base_users": 1},
```

Comment:
Not necessary to change but for future use: Instead of splitting the values and ids you can use the newer `pytest.param()` style.
Example:
https://github.com/raiden-network/raiden/blob/aa9fdbd2dedd379e175e1329c933586f4ddaa0d8/raiden/tests/unit/test_utils.py#L51-L57